src/store/actions/transactions.test.js

```diff
@@ -35,6 +52,10 @@ describe('actions: transactions', () => {
     },
   });
 
+  // afterEach(() => {
```

Comment:
Remove the commented code if not required, same applies to below code as well.

Reply:
Done. Will reflect on my next push